runatlantis.io/docs/server-configuration.md

```diff
@@ -447,6 +447,16 @@ and set `--autoplan-modules` to `false`.
 
   This is useful when running multiple Atlantis servers against a single repository.
 
+### `--hide-unchanged-plan-comments`
+  ```bash
+  atlantis server --hide-unchanged-plan-comments
```

Comment:
Let me ping the @runatlantis/maintainers and other folks @marcusramberg @krrrr38 @Fabianoshz @zhuoyang how they feel about this flag name.

Always good to get more eyes on new features.

Reply:
name sounds good to me